src/time.cpp

```diff
@@ -32,9 +32,33 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #include "libtorrent/aux_/time.hpp"
 
+#include <chrono>
+
 namespace libtorrent { namespace aux {
 
 	time_point time_now() { return clock_type::now(); }
 	time_point32 time_now32() { return time_point_cast<seconds32>(clock_type::now()); }
 
+	// for simplying implementation
+	using std::chrono::system_clock;
+
+	time_t to_time_t(const time_point32 tp)
```

Comment:
would you mind adding a comment suggesting that in C++20 this can be replaced by `std::chrono::clock_cast`? (which I believe is the case)